mods/mtg_craftguide/locale/mtg_craftguide.es.tr

```diff
@@ -0,0 +1,57 @@
+# textdomain: mtg_craftguide
+
+Any coal=carbón
+Any sand=arena
+Any wool=lana
+Any stick=palitos
+Any vessel=recipiente
+Any wood planks=tablas de madera
+Any kind of stone block=derivado de bloque de piedra
+Any red flower=flor roja
+Any blue flower=flor azul
+Any black flower=flor negra
+Any green flower=flor verde
+Any white flower=flor blanca
+Any orange flower=flor naranja
+Any violet flower=flor violeta
+Any yellow flower=flor amarilla
+Any red dye=tinte rojo
+Any blue dye=tinte azul
+Any cyan dye=tinte cian
+Any grey dye=tinte gris
+Any pink dye=tinte rosa
+Any black dye=tinte negro
+Any brown dye=tinte marrón
+Any green dye=tinte verde
+Any white dye=tinte blanco
+Any orange dye=tinte naranja
+Any violet dye=tinte violeta
+Any yellow dye=tinte amarillo
+Any magenta dye=tinte magenta
+Any dark grey dye=tinte gris oscuro
+Any dark green dye=tinte verde oscuro
```

Comment:
How come these do not start with a capital letter? It might look better in-game if everything is consistent.